### DISCUSSION TITLE
Welcome to getoptions Discussions!

### DISCUSSION THREAD

**Discussion by ko1nksm** [ko1nksm/getoptions#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by marcellodesales** [comment]
@ko1nksm That's an awesome lib!!! Thanks a lot for your work!I have a question!!! Please help me implement commands and subcommands that share the same name! https://github.com/ko1nksm/getoptions/discussions/16
